docs/getting-started-guides/docker-multinode.md

```diff
@@ -53,7 +49,9 @@ Please install Docker 1.6.2 or Docker 1.7.1.
 
 ## Prerequisites
 
-1. You need a machine with docker of right version installed.
+The only thing you need is a machine with **Docker 1.8.x or higher**
+
+It might work with older docker versions, but that isn't supported.
```

Comment:
Kubernetes itself doesn't require docker 1.8. If we know that this particular setup will not work with 1.7.0, then believe we should just write "1.7.1+".